tools/Google.Cloud.Docs.Snippets/FaqSnippets.cs

```diff
@@ -37,6 +38,7 @@ public void Emulator()
             {
                 Console.WriteLine(topic.Name);
             }
+            // [START pubsub_use_emulator]
```

Comment:
Presumably this is intended to be END rather than START

Reply:
Yes, thanks!

Reply:
I've merged a tool change, so once this is fixed I'll be able to merge.